common/src/main/kotlin/org/jetbrains/kotlinconf/presentation/SessionDetailsPresenter.kt

```diff
@@ -21,15 +21,15 @@ class SessionDetailsPresenter(
 
     fun onCreate() {
         refreshDataFromRepo()
-//        repository.addUpdateListener(onRefreshListener)
+        repository.onRefreshListeners += onRefreshListener
```

Comment:
Please consider avoiding comments with code

Reply:
I always avoid them.

Reply:
I removed comment here. I don't know who introduced it. Can we resolve this?